### PR TITLE
Bump to jGit 5.9.0.202009080501-r

### PR DESCRIPTION
**Pull request by gastaldi**
It also adds the missing GA in the dependabot descriptor

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ updates:
       - dependency-name: org.freemarker:freemarker
       - dependency-name: org.eclipse.jgit:org.eclipse.jgit
       - dependency-name: org.eclipse.jgit:org.eclipse.jgit.http.server
+      - dependency-name: org.eclipse.jgit:org.eclipse.jgit.ssh.jsch
```
Comment (gsmet) [inline .github/dependabot.yml:23]: Shouldn't we just have `org.eclipse.jgit:*`? I tested wildcard and they are working fine.

Comment (gastaldi) [reply .github/dependabot.yml]: Sure, done 👍🏻

**Review by geoand — APPROVED** [review]

**Comment by geoand** [comment]
We should start thinking about moving extensions like this out of the core

**Comment by gastaldi** [comment]
> We should start thinking about moving extensions like this out of the core

+1, I'd love to do for this particular extension. Maybe I'll also adopt it as the guinea pig for enhancements in Quarkiverse 😉

**Comment by geoand** [comment]
💯

**Comment by famod** [comment]
Btw, JGit 5.10.0 was released earlier today: https://projects.eclipse.org/projects/technology.jgit/releases/5.10.0/

**Comment by gastaldi** [comment]
@famod are you sure? that page says that 5.10 is going to be released on Wednesday, December 16, 2020

**Comment by famod** [comment]
@gastaldi Yes, a bit strange but then again: https://github.com/vackosar/gitflow-incremental-builder/pull/286

**Comment by gastaldi** [comment]
Heh, well, let's see if dependabot works now 😅

**Pull request merged**